### DISCUSSION TITLE
Setting minimum read count

### DISCUSSION THREAD

**Discussion by sdwien** [GoekeLab/m6anet#32] [Q&A]
Dear m6anet Team,

would you recommend setting the "Minimum read counts per gene" argument to something like 10 or 100 for a first try, in order to see real confident positions? My doubt here is that I might lose a lot of sites, since with direct RNA sequencing, I have very low read numbers for most of the genes. 

Many thanks for your thoughts.
Best regards, Sophia

**Comment by chrishendra93** [comment]
Hi Sophia,

Sorry for the delay in my response, I was on leave previously and was only back recently. Currently m6Anet requires 20 reads to make inference, hence the minimum number of read counts per gene. Also this argument helps to speed the preprocessing.

I am planning to add more features in the near future to reduce this requirement and give more flexibility with regards to the output, i.e, allow some individual read m6A probability even when inference at site level is not possible. 

Regards

Christopher Hendra

**Reply by SamuelErnestRoss** [reply]
Hello Christopher,

are there any updates on these potential future features to look at m6A levels on regions with less than 20 reads? 

Thanks
Sam

**Reply by chrishendra93** [reply]
Hi Sam,

Technically speaking this can be done - read level probability can be inferred for each read but the site level probability might not be as reliable. The size of the output might increase by a lot though, do you have any suggestion on what do you think will be useful for this?
